app/views/artefacts/form/_tags.html.erb

```diff
@@ -1,56 +1,5 @@
 <%- unless f.object.app_without_tagging_support? %>
   <%- if f.object.tagging_migrated? %>
```

Comment:
gotcha. So should I remove any reference to 'migrated apps' such as: https://github.com/alphagov/panopticon/search?utf8=%E2%9C%93&q=migrated&type=Code